### PR TITLE
cnf network: fix-dpdk-test-case

### PR DESCRIPTION
**Pull request by gkopels**

### CODE REVIEW

File: tests/cnf/core/network/sriov/tests/qinq.go [hunk 1]
```diff
@@ -69,16 +70,20 @@ var _ = Describe("QinQ", Ordered, Label(tsparams.LabelQinQTestCases), ContinueOn
...
+		intelDeviceIDE710           = "158b"
 		testCmdNet2                 = []string{"bash", "-c", "sleep 5; testcmd -interface net2 -protocol tcp " +
 			"-port 4444 -listen"}
 		testCmdNet2Net3 = []string{"bash", "-c", "sleep 5; testcmd -interface net2 -protocol tcp " +
 			"-port 4444 -listen & testcmd -interface net3 -protocol tcp -port 4444 -listen"}
 		testCmdBond0 = []string{"bash", "-c", "sleep 5; testcmd -interface bond0.100 -protocol tcp " +
 			"-port 4444 -listen"}
-		tcpDumpNet1CMD              = []string{"bash", "-c", "tcpdump -i net1 -e > /tmp/tcpdump"}
+		tcpDumpNet1CMD = []string{"bash", "-c", "tcpdump -i net1 -e > /tmp/tcpdump"}
+
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/qinq.go:81]: remove space

File: tests/cnf/core/network/sriov/tests/qinq.go [hunk 2]
```diff
@@ -712,6 +706,19 @@ func defineAndCreateSrIovNetworkWithQinQ(srIovNetwork, resName, vlanProtocol str
 		"Fail to pull NetworkAttachmentDefinition")
 }
 
+func defineAndCreateSrIovNetworkClientDPDK(srIovNetwork, resName string) {
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/qinq.go:709]: srIovNetworkName ?

File: tests/cnf/core/network/sriov/tests/qinq.go [hunk 3]
```diff
@@ -712,6 +706,19 @@ func defineAndCreateSrIovNetworkWithQinQ(srIovNetwork, resName, vlanProtocol str
...
+	srIovNetworkObject, err := sriov.NewNetworkBuilder(
+		APIClient, srIovNetwork, NetConfig.SriovOperatorNamespace, tsparams.TestNamespaceName, resName).Create()
+	Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("Failed to create sriov network %s", err))
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/qinq.go:712]: remove fmt.Sprintf and err

File: tests/cnf/core/network/sriov/tests/qinq.go [hunk 4]
```diff
@@ -814,6 +821,7 @@ func readAndValidateTCPDump(clientPod *pod.Builder, testCmd []string, pattern st
 	By("Start to capture traffic on the promiscuous client")
 
 	output, err := clientPod.ExecCommand(testCmd)
+	fmt.Println(output.String())
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/qinq.go:824]: remove ?

File: tests/cnf/core/network/sriov/tests/qinq.go [hunk 5]
```diff
@@ -956,16 +966,15 @@ func defineAndCreateClientDPDKPod(
 	Expect(err).ToNot(HaveOccurred(), "Fail to define client dpdk container")
 
 	dpdkPod, err := pod.NewBuilder(APIClient, podName, tsparams.TestNamespaceName,
-		NetConfig.DpdkTestContainer).WithSecondaryNetwork(serverPodNetConfig).
+		NetConfig.DpdkTestContainer).WithSecondaryNetwork(serverPodNetConfig).WithPrivilegedFlag().
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/qinq.go:969]: Why do you need WithPrivilegedFlag() if you created corev1.SecurityContext?

File: tests/cnf/core/network/sriov/tests/qinq.go [hunk 6]
```diff
@@ -524,37 +522,33 @@ var _ = Describe("QinQ", Ordered, Label(tsparams.LabelQinQTestCases), ContinueOn
...
 			reportxml.ID("72638"), func() {
-				By("Define and create a container in promiscuous mode")
-				tcpDumpContainer := createPromiscuousClient(workerNodeList[0].Definition.Name,
-					tcpDumpNet1CMD)
+				testOutPutString := tcpDumpDot1QDPDKOutput
+				if sriovDeviceID == intelDeviceIDE710 {
+					vlan, err := strconv.Atoi(NetConfig.VLAN)
+					Expect(err).ToNot(HaveOccurred(), "Failed to convert VLAN value")
+					testOutPutString = fmt.Sprintf("(ethertype 802\\.1Q \\(0x8100\\)).*?(vlan %d)", vlan)
+					fmt.Println(testOutPutString)
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/qinq.go:542]: Just print??

File: tests/cnf/core/network/sriov/tests/qinq.go (hunk 1 above)
Comment (gkopels) [reply tests/cnf/core/network/sriov/tests/qinq.go:81]: removed

File: tests/cnf/core/network/sriov/tests/qinq.go (hunk 2 above)
Comment (gkopels) [reply tests/cnf/core/network/sriov/tests/qinq.go:709]: Changed to srIovNetworkName

File: tests/cnf/core/network/sriov/tests/qinq.go (hunk 3 above)
Comment (gkopels) [reply tests/cnf/core/network/sriov/tests/qinq.go:712]: changed

File: tests/cnf/core/network/sriov/tests/qinq.go (hunk 4 above)
Comment (gkopels) [reply tests/cnf/core/network/sriov/tests/qinq.go:824]: removed

File: tests/cnf/core/network/sriov/tests/qinq.go (hunk 6 above)
Comment (gkopels) [reply tests/cnf/core/network/sriov/tests/qinq.go:542]: removed

File: tests/cnf/core/network/sriov/tests/qinq.go (hunk 5 above)
Comment (gkopels) [reply tests/cnf/core/network/sriov/tests/qinq.go:969]: removed

**Review by evgenLevin — APPROVED** [review]
/lgtm

**Review by ajaggapa — APPROVED** [review]
/lgtm

**Pull request merged**